### PR TITLE
Taxa plots

### PR DESCRIPTION
**Pull request by adamrp**
Basics are in place.
- Where do I place the taxa summary txt files?  Clearly, img/taxa_summaries isn't ideal since these aren't images, but I didn't know where to put them for now.
- It's not exactly pretty, but I don't know if it really needs to be...

### CODE REVIEW

**Comment by adamrp** [comment]
Should be ready to merge if code looks okay (btw: the code is kinda ugly, but not sure if it can be made pretty)

**Comment by wasade** [comment]
seems fine to me, where there any other stylistic things that were worth addressing or should this get merged?

**Comment by adamrp** [comment]
Several people have said that alternating row colors would be good, but Rob did say that it looked good (enough) to go right now.  So adding alternating row colors is something I can do in a later pull request if we want.

**Comment by wasade** [comment]
that's fine. would be good to get this up and have jeff send out something

**Comment by wasade** [comment]
shall i merrrge?

**Comment by adamrp** [comment]
The only other thing is that I didn't know where in the file hierarchy to
place the "taxa_plots" directory, where the text files are actually stored.
 Right now, it's in the img/taxa_plots directory, but I am not sure it's
ideal to have them under the img directory since they are not images.  Was
thinking of moving them up a directory (so the taxa_plots directory is on
the same level as img), but wasn't sure if maybe we wanted to rework any of
that structure.

**Comment by wasade** [comment]
No strong feelings, suggest minimizing effort

**Comment by ElDeveloper** [comment]
:+1: on moving `taxa_plots` up a directory. And also renaming the directory to `taxa_summaries` as these are really not plots.

**Comment by wasade** [comment]
anything blocking this or is it basically good to go?

**Comment by ElDeveloper** [comment]
AFAIK @adamrp was going to move the taxa_plots directory up right?  And also renaming the directory to taxa_summaries as these are really not plots.

**Comment by adamrp** [comment]
Yeah, I can do that.

**Comment by wasade** [comment]
Just realized, we should probably make sure to either mention that these data are rarified to 1000seqs per sample (confusing) or to make sure they're in relative abundance. I don't know what was being returned from the results processing code off hand though

**Comment by adamrp** [comment]
Just tested, and this should be good to go now.  Can someone else test?  This is deployed on webdev.  The participant "foo" has two samples associated, one that has a taxa summary available (000000001), and the others do not.

File: www/americangut/taxa_summary.psp [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<%
+from os.path import join
```
Comment (ElDeveloper) [inline www/americangut/taxa_summary.psp:11]: Could you add the credit strings to this file? i. e. author, maintainer, credits, e-mail, etc.

File: www/americangut/taxa_summary.psp [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    #END INDENT
+%>
+
+<div class="content">
+    <div class="red">
+        Note: Where there are blanks in the table below, the taxonomy could not be resolved in finer detail.
+        <br>
+        Note: The percentages listed represent the relative abundance of each taxon.
+    </div>
+    <a href="taxa_summaries/<%=barcode%>.txt">Download the table</a>
```
Comment (ElDeveloper) [inline www/americangut/taxa_summary.psp]: Not sure if this is worth to add but there's a `download` tag that you can add here that will force a "Save As ..." window to show up when this link is clicked. See:

http://stackoverflow.com/questions/11353425/force-a-browser-to-save-file-as-after-clicking-link

**Comment by ElDeveloper** [comment]
It works fine, in the test that I made and is likely ready to be merged after that minor comment I made.

**Comment by adamrp** [comment]
Thanks, Yoshiki -- good suggestions.  I have made the requested modifications.

**Comment by ElDeveloper** [comment]
Looks good, thanks!

**Pull request merged**